### ISSUE TITLE
Persistent Data Path config and config file placements are mismatched, resulting in config file not being used.

### ISSUE THREAD

**Issue by Frostea** [Mephi00/v-rising-wine-docker-image#3]
In entrypoint.sh, `-persistentDataPath Z:\\saves` is specified. This means that the server will take Z:\\saves as the config/save folder.
This folder will have a structure like this:

> 
> Z:\\saves
> Z:\\saves\\Saves\\v1\\<list_of_save_folders>
> Z:\\saves\\Settings\\ServerGameSettings.json
> Z:\\saves\\Settings\\ServerHostSettings.json

Currently in entrypoint.sh, the config files `ServerGameSettings.json` and `ServerHostSettings.json` are copied into the default config location. However, by specifying `-persistentDataPath`, the default config location will not be used.

**Comment by zhixXiang** [comment]
You are right, put the Settings in the /saves file will work.

**Comment by Mephi00** [comment]
The default location is still used if there are no setting files in the specified persistenceDataPath.

I will however adjust the documentation and entrypoint script accordingly once I have the option to do so.
Thank you for mentioning this.

**Comment by Mephi00** [comment]
I have added the functionality and documentation, but refrained from writing the settings into /saves/Settings. I wanted to update the settings, when the environment variables change. This action would, however, overwrite the already present files in /saves/Settings. To avoid overwriting custom configurations in /saves/Settings, I opted to not use the folder unless it is already present, in which case a change of env vars does not cause a change in configuration. This behavior has been documented accordingly. 

I hope this provides closure and a good explanation on why I chose not to use /saves/Settings as the default settings folder.
As stated above, the default settings path is used, when the folder /saves/Settings is not present.
